### PR TITLE
Add io.github.sigmasd.share

### PR DESCRIPTION
**Pull request by sigmaSd**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly. : An application that share files
- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements#control-over-domain
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: io.github.sigmasd.share.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+app-id: io.github.sigmasd.share
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: share
+
+finish-args: 
+  - --share=network
```
Comment (bbhtt) [inline io.github.sigmasd.share.yml:10]: Why do you need network access?

**Comment by bbhtt** [comment]
bot, build io.github.sigmasd.share

File: io.github.sigmasd.share.yml (hunk 1 above)
Comment (sigmaSd) [reply io.github.sigmasd.share.yml:10]: The app :
- spawns a local server # needs network access for this
- accepts file drop
- when the user scans the qr code, he gets the server address

**Comment by sigmaSd** [comment]
bot, build io.github.sigmasd.share

File: io.github.sigmasd.share.yml (hunk 1 above)
Comment (bbhtt) [reply io.github.sigmasd.share.yml:10]: Alright

**Comment by bbhtt** [comment]
bot, build io.github.sigmasd.share

**Pull request closed without merging**